internal/flavors/benchmark/eks_test.go

```diff
@@ -59,36 +58,18 @@ func TestEKS_Initialize(t *testing.T) {
 			name:    "nothing initialized",
 			wantErr: "uninitialized",
 		},
-		{
-			name:           "uninitialized aws provider",
-			clientProvider: mockKubeClient(nil),
-			cfg:            awsCfg,
-			wantErr:        "aws config provider is uninitialized",
-		},
-		{
-			name:                   "aws error",
-			awsCfgProvider:         mockAwsCfg(errors.New("some error")),
-			clientProvider:         mockKubeClient(nil),
-			awsIdentityProvider:    mockAwsIdentityProvider(errors.New("not this error")),    // ineffectual
-			awsMetadataProvider:    mockMetadataProvider(errors.New("not this error")),       // ignored
-			eksClusterNameProvider: mockEksClusterNameProvider(errors.New("not this error")), // ignored
-			cfg:                    awsCfg,
-			wantErr:                "some error",
-		},
 		{
 			name:                   "aws identity provider error",
-			awsCfgProvider:         mockAwsCfg(nil),
 			awsIdentityProvider:    mockAwsIdentityProvider(errors.New("some error")),
 			clientProvider:         mockKubeClient(nil),
-			awsMetadataProvider:    mockMetadataProvider(errors.New("not this error")),       // ignored
+			awsMetadataProvider:    mockMetadataProvider(nil),
```

Comment:
this needs to not err because EKS now sets the region itself via the metadata provider